canonical-web-and-design/cookie-policy
# Refactor to output both ESM and IIFE targets

Note: Due to the updated function call signature, this is a breaking change.

## Done

- Updated package to export both ESM nd IIFE targets

## QA

- Pull down this feature branch
- Open `index.html` in a browser
- Use `npm link` to check this package can be consumed by JS application (e.g. use create-react-app to spin up sample app, then `npm i` it)
